From d926a110ceaae08cb3844c2877a924ea4c6fa5f7 Mon Sep 17 00:00:00 2001
From: Bare <git@barebrowser.org>
Date: Sun, 11 Oct 2026 09:21:29 +0000
Subject: [PATCH] Rename the browser in four strings new in 156

156 adds four strings that name Chrome and that a Bare user can reach: the
Tabs setting that keeps custom tabs out of the browser, its summary, the
Ctrl+Q entry in the keyboard shortcut list, and the notification offered after
a trusted web app is uninstalled. They now say Bare, like the rest of the
interface. The translator notes are left as they are.
---
 .../browser/ui/android/strings/android_chrome_strings.grd | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/chrome/browser/ui/android/strings/android_chrome_strings.grd b/chrome/browser/ui/android/strings/android_chrome_strings.grd
index d781a0a527a..f794dc5c043 100644
--- a/chrome/browser/ui/android/strings/android_chrome_strings.grd
+++ b/chrome/browser/ui/android/strings/android_chrome_strings.grd
@@ -212,10 +212,10 @@ CHAR_LIMIT guidelines:
         Advanced
       </message>
       <message name="IDS_CCT_ALWAYS_OPEN_IN_BROWSER_TITLE" desc="Title for preference toggle in Settings to always open Custom Tabs in Chrome browser. [CHAR_LIMIT=40]">
-        Always open custom tabs in Chrome
+        Always open custom tabs in Bare
       </message>
       <message name="IDS_CCT_ALWAYS_OPEN_IN_BROWSER_SUMMARY" desc="Summary for preference toggle in Settings to always open Custom Tabs in Chrome browser. [CHAR_LIMIT=60]">
-        Open web pages in Chrome browser instead of custom tabs
+        Open web pages in Bare instead of custom tabs
       </message>
       <message name="IDS_PREFS_NEW_LABEL" desc="The label displayed next to the title of newly added preferences.">
         <ph name="PREF_TITLE">%1$s</ph>  <ph name="BEGIN_NEW">&lt;new&gt;</ph>New<ph name="END_NEW">&lt;/new&gt;</ph>
@@ -6098,7 +6098,7 @@ To change this setting, <ph name="BEGIN_LINK">BEGIN_LINK</ph>delete the Chrome d
         App was uninstalled
       </message>
       <message name="IDS_TWA_POST_UNINSTALL_NOTIFICATION_TEXT" desc="Text for the notification sent after a TWA app is uninstalled">
-        Tap to delete browsing data in Chrome
+        Tap to delete browsing data in Bare
       </message>
 
       <message name="IDS_WEBAPP_TAP_TO_COPY_URL" desc="Message on the notification that indicates that taping it will copy a Web App's URL into the clipboard.">
@@ -6256,7 +6256,7 @@ To change this setting, <ph name="BEGIN_LINK">BEGIN_LINK</ph>delete the Chrome d
         Toggle caret browsing
       </message>
       <message name="IDS_KEYBOARD_SHORTCUT_QUIT_CHROME" desc="A text label that appears next to the keyboard shortcut that will quit Chrome. The shortcut description is shown in a system dialog along with all other supported shortcuts. [CHAR_LIMIT=55]">
-        Quit Chrome
+        Quit Bare
       </message>
       <message name="IDS_CONFIRM_TO_QUIT_TOAST" desc="Toast message instructing the user to hold Ctrl+Q to quit Chrome.">
         Hold Ctrl+Q to quit
